.lintstagedrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "*.{js,vue}": ["npm run lint"]
```

Comment:
can we get it to run on our component tests as well? `*.cy.js`?

Reply:
I believe it does as is